manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -511,12 +518,20 @@ func (r *FybrikApplicationReconciler) ValidateAssetResponse(response *datacatalo
 		datasetID, allErrs)
 }
 
+// constructDataInfo collects the following information about the asset:
+// - asset metadata
+// - governance actions to be performed on the data
+// - potential governance actions in case of caching the asset in a specific location
+// - decisions after evaluating config policies
+// The function returns an error received whilw communication with connectors or evaluating policies
```

Comment:
whilw?
The function returns an error received (from, by?) communication with connectors or evaluating policies

Reply:
Fixed